src/ctrl.c

```diff
@@ -8,18 +8,85 @@
 #include "conn_sock.h"
 #include "cmsg.h"
 #include "cli.h" // opt_bundle_path
+#include "seccomp_notify.h"
 
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <termios.h>
 
+#include <seccomp.h>
+#include <linux/seccomp.h>
+#include <sys/sysmacros.h>
+#include <sys/wait.h>
+#include <sys/mount.h>
+#include <signal.h>
+
+static struct seccomp_notify_context_s *seccomp_notify_ctx;
+
 static void resize_winsz(int height, int width);
 static gboolean read_from_ctrl_buffer(int fd, gboolean (*line_process_func)(char *));
 static gboolean process_terminal_ctrl_line(char *line);
 static gboolean process_winsz_ctrl_line(char *line);
 static void setup_fifo(int *fifo_r, int *fifo_w, char *filename, char *error_var_name);
 
+static void cleanup_seccomp_plugins()
```

Comment:
why put these in this file? we aren't doing anything with the ctrl file afaict

Reply:
moved to seccomp_notify.c